onnx/onnx.in.proto

```diff
@@ -379,8 +379,23 @@ message TensorShapeProto {
       int64 dim_value = 1;
       string dim_param = 2;   // namespace Shape
     };
+    // Standard denotation can optionally be used to denote tensor
+    // dimensions with standard semantic descriptions to ensure
+    // that operations are applied to the correct axis of a tensor.
+    optional string standard_denotation = 3;
   };
   repeated Dimension dim = 1;
+
+  // A set of pre-defined constants to be used as values for
```

Comment:
this needs to live in a separate message (as it's the fake message just for grouping purposes for constants). Otherwise those fields would be present in every TensorShapeProto instance.